### PR TITLE
Add profile backgrounds

### PR DESCRIPTION
**Pull request by tsunyoku**

### CODE REVIEW

File: app/adapters/s3.py [hunk 1]
```diff
@@ -47,4 +51,10 @@ async def download(file_name: str, folder: str) -> bytes | None:
...
         return None
 
-    return await response["Body"].read()
+    body = await response["Body"].read()
+    content_type = response["ContentType"]
+
+    return {
+        "body": body,
+        "content_type": content_type,
+    }
```
Comment (cmyui) [inline app/adapters/s3.py:60]: hmm checked s3 docs, the possible return fields are 
```js
{
    'Body': StreamingBody(),
    'DeleteMarker': True|False,
    'AcceptRanges': 'string',
    'Expiration': 'string',
    'Restore': 'string',
    'LastModified': datetime(2015, 1, 1),
    'ContentLength': 123,
    'ETag': 'string',
    'ChecksumCRC32': 'string',
    'ChecksumCRC32C': 'string',
    'ChecksumSHA1': 'string',
    'ChecksumSHA256': 'string',
    'MissingMeta': 123,
    'VersionId': 'string',
    'CacheControl': 'string',
    'ContentDisposition': 'string',
    'ContentEncoding': 'string',
    'ContentLanguage': 'string',
    'ContentRange': 'string',
    'ContentType': 'string',
    'Expires': datetime(2015, 1, 1),
    'WebsiteRedirectLocation': 'string',
    'ServerSideEncryption': 'AES256'|'aws:kms'|'aws:kms:dsse',
    'Metadata': {
        'string': 'string'
    },
    'SSECustomerAlgorithm': 'string',
    'SSECustomerKeyMD5': 'string',
    'SSEKMSKeyId': 'string',
    'BucketKeyEnabled': True|False,
    'StorageClass': 'STANDARD'|'REDUCED_REDUNDANCY'|'STANDARD_IA'|'ONEZONE_IA'|'INTELLIGENT_TIERING'|'GLACIER'|'DEEP_ARCHIVE'|'OUTPOSTS'|'GLACIER_IR'|'SNOW'|'EXPRESS_ONEZONE',
    'RequestCharged': 'requester',
    'ReplicationStatus': 'COMPLETE'|'PENDING'|'FAILED'|'REPLICA'|'COMPLETED',
    'PartsCount': 123,
    'TagCount': 123,
    'ObjectLockMode': 'GOVERNANCE'|'COMPLIANCE',
    'ObjectLockRetainUntilDate': datetime(2015, 1, 1),
    'ObjectLockLegalHoldStatus': 'ON'|'OFF'
}
```

no AI for now - but eventually we should aim to correctly support this - if we are going with a Response object

File: app/api/v1/avatars.py [hunk 2]
```diff
@@ -15,12 +15,12 @@ async def get_avatar(file_path: str):
...
         # fallback to a default avatar so we don't need to
         # duplicate the default avatar for all of our userbase
-        content = await s3.download(settings.DEFAULT_AVATAR_FILENAME, "avatars")
-        if content is None:
+        download_response = await s3.download(settings.DEFAULT_AVATAR_FILENAME, "avatars")
+        if download_response is None:
             return Response(status_code=404)
 
-    return Response(content=content, media_type="image/png")
+    return Response(content=download_response["body"], media_type="image/png")
```
Comment (cmyui) [inline app/api/v1/avatars.py]:
```suggestion
    return Response(content=download_response["body"], media_type=download_response["content_type"])
```

File: app/adapters/s3.py (hunk 1 above)
Comment (tsunyoku) [reply app/adapters/s3.py:60]: yeah, i didn't think there was much reason to include all of the possible fields but if you want to then i can

File: app/api/v1/avatars.py (hunk 2 above)
Comment (tsunyoku) [reply app/api/v1/avatars.py]: funnily enough i did that initially but then removed it, above in the code it pretty much enforces png so i didn't see the point (hanayo also ensures png before upload)

**Review by cmyui — APPROVED** [review]

File: app/api/v1/avatars.py (hunk 2 above)
Comment (cmyui) [reply app/api/v1/avatars.py]: minor but feels more flexible

Comment (tsunyoku) [reply app/api/v1/avatars.py]: yea sure i'll apply it again

**Pull request merged**